docs/src/osdk/reference/manifest.md

```diff
@@ -32,58 +32,103 @@ Below, you will find a comprehensive version of
 the available configuration options in the manifest.
 
 ```toml
-kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <1>
-init_args = ["sh", "-l"] # <2>
-initramfs="./build/initramfs.cpio.gz" # <3>
-
-[boot]
-loader = "grub" # <4>
-protocol = "multiboot2" # <5>
-grub-mkrescue = "/usr/bin/grub-mkrescue" # <6>
-ovmf = "/usr/bin/ovmf" # <7>
-
-[qemu]
-path = "/usr/bin/qemu-system-x86_64" # <8>
-machine = "q35" # <9>
-args = [ # <10>
-    "-enable-kvm",
-    "-m 2G", 
-    "-device virtio-keyboard-pci,disable-legacy=on,disable-modern=off"
-] 
+[project] # <1>
+type = "kernel"
+
+[run] # <2>
+kcmd_args = ["init=/bin/busybox", "path=/usr/local/bin"] # <3>
+init_args = ["sh", "-l"] # <4>
+initramfs="./build/initramfs.cpio.gz" # <5>
+bootloader = "grub" # <6>
+boot_protocol = "multiboot2" # <7>
+grub-mkrescue = "/usr/bin/grub-mkrescue" # <8>
+ovmf = "/usr/bin/ovmf" # <9>
+qemu_path = "/usr/bin/qemu-system-x86_64" # <10>
+qemu_args = [ # <11>
+    "-machine q35,kernel-irqchip=split",
+    "-cpu Icelake-Server,+x2apic",
+    "-m 2G",
+]
+
+[test] # <2>
+bootloader = "qemu"
+qemu_args = [ # <10>
+    "-machine q35,kernel-irqchip=split",
+    "-cpu Icelake-Server,+x2apic",
+    "-m 2G",
+]
+
+['cfg(arch="x86_64", schema=microvm)'.run] # <12>
+bootloader = "qemu"
```

Comment:
Maybe it isn't a typo. I changed `boot_loader` to `bootloader` since I see it as a whole word the most often.

https://en.wikipedia.org/wiki/Bootloader

But boot loader is also fine. I think it's a problem of habits.

Reply:
Ok, I should have been more accurate. So here `QEMU` is considered a bootloader? This is really weird to me.

Reply:
> Ok, I should have been more accurate. So here `QEMU` is considered a bootloader? This is really weird to me.

Yes QEMU is a bootloader. It is capable to load Multiboot or PVH kernels.